### PR TITLE
remove some test users; add dedicated cypress test users

### PR DESCRIPTION
**Pull request by braxex**
what if this?

by assigning them to DC we can probably avoid lots of the mess created in envs.

### CODE REVIEW

**Comment by braxex** [comment]
@tblackwe @BearHanded thoughts?

**Comment by braxex** [comment]
> @tblackwe @BearHanded thoughts?

What do you think?

**Review by tblackwe — COMMENTED** [review]
Approach looks good to me, but I am unfamiliar with how our accounts are administered.

How does the Incognito password tie into these 2 new users?

File: services/ui-auth/libs/users.json [hunk 1]
```diff
@@ -249,48 +249,44 @@
...
         "Value": "true"
       },
       {
         "Name": "custom:cms_roles",
-        "Value": "mdctmcr-state-user"
-      },
-      {
-        "Name": "custom:cms_state",
-        "Value": "CA"
+        "Value": "mdctmcr-bor"
```
Comment (tblackwe) [inline services/ui-auth/libs/users.json:272]: Seeking to understand: how this value used?

**Review by tblackwe — APPROVED** [review]

**Review by karla-vm — APPROVED** [review]

**Pull request merged**